client.lua

```diff
@@ -808,6 +841,35 @@ local function SpawnPed(data)
 					TaskStartScenarioInPlace(spawnedped, v.scenario, 0, true)
 				end
 
+				if v.pedrelations and type(v.pedrelations.groupname) == 'string' then
+					local pedgrouphash = joaat(v.pedrelations.groupname)
+
+					if not DoesRelationshipGroupExist(pedgrouphash) then
+						AddRelationshipGroup(v.pedrelations.groupname)
+					end
+
+					SetPedRelationshipGroupHash(spawnedped, pedgrouphash)
+					if v.pedrelations.toplayer then
+						SetRelationshipBetweenGroups(v.pedrelations.toplayer, pedgrouphash, joaat('PLAYER'))
+					end
+
+					if v.pedrelations.toowngroup then
+						SetRelationshipBetweenGroups(v.pedrelations.toowngroup, pedgrouphash, pedgrouphash)
+					end
+				else
+					if Config.Debug then print(v.pedrelations.groupname .. ' is not a string') end
```

Comment:
Please make this throw an error by using `error()`

Reply:
All done